biz_content/forms.py

```diff
@@ -52,3 +52,9 @@ class PermitStatusForm(forms.Form):
     permit_id = forms.CharField(required=True,
                                 label="Permit ID",
                                 help_text="Enter your Permit Application ID")
+
+
+class BizLicenseStatusForm(forms.Form):
+    cu_id = forms.CharField(required=True,
```

Comment:
This need more validation to confirm its actually of the correct form.  Also we need to validate its not attempting anything fishy since we use it build a url in the view